src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/.editorconfig

```diff
@@ -1,5 +1,4 @@
 # Editor configuration, see http://editorconfig.org
-root = true
```

Comment:
@javiercn - is this actually part of the template content, and should be left alone?

Reply:
Yes, we should leave this alone